go/versions/test.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
This is the sample data for the manifest. It's like a package.json lite and versions contain where to find it, build number and version.

If it's a release version I just mark it with build_number:0 which means it's the first and only build.